Makefile

```
@@ -3,11 +3,37 @@ BINARY_NAME=netronome
+
+# Private repo holding the premium theme CSS. Absent in forks and source builds,
+# which ship the default theme only.
+THEMES_REPO=github.com/autobrr/qui-premium-themes
+THEMES_DIR=web/src/themes/premium
 
 all: build
 
-build:
+# Fetch premium themes from the private repository. Requires THEMES_REPO_TOKEN;
+# without it the build succeeds and simply ships no premium themes.
+themes-fetch:
```

checkmake (0.3.2)

[warning] Target body for "themes-fetch" exceeds allowed length of 5 lines (13).

(maxbodylength)